cmdstanpy/utils.py

```diff
@@ -130,12 +133,8 @@ def check_csv(path: str) -> Dict:
         draws_spec = int(dict['num_samples'])
     else:
         draws_spec = 1000
-    if 'num_warmup' in dict:
-        num_warmup = int(dict['num_warmup'])
-    else:
-        num_warmup = 1000
-    if 'save_warmup' in dict and dict['save_warmup'] == '1':
-        draws_spec = draws_spec + num_warmup
+    if 'thin' in dict:
+        draws_spec = int(math.ciel(draws_spec / dict['thin']))
```

Comment:
math.ceil

int is not needed, math.ceil returns an integer

is this true?

    math.ceil(draws_spec / dict['thin'])

vs

    draws_spec // dict['thin']

    ceil(1000/3) == 334
    1000 // 3 == 333

Reply:
if you run CmdStan with `thin=3`, you will get 334 draws.

Reply:
Should be `math.ceil` (notice the typo)

Reply:
cannot spell the word "ceiling"  - (in any language).